run_master.sh

```diff
@@ -0,0 +1 @@
+docker run -p 5555:5555 -v /var/run/docker.sock:/var/run/docker.sock -d kraleppa/data-farmer-master:0.0.2
```

Comment:
and let's run it like `./run_master.sh 0.0.2` 😄 
```suggestion
docker run -p 5555:5555 -v /var/run/docker.sock:/var/run/docker.sock -d kraleppa/data-farmer-master:$1
```